homeassistant/components/renson/const.py

```diff
@@ -0,0 +1,4 @@
+"""Constants for the Renson integration."""
+
+DOMAIN = "renson"
+CONCENTRATION_PARTS_PER_CUBIC_METER = "ppm"
```

Comment:
Can we use our common constant for ppm?

https://github.com/home-assistant/core/blob/a6d6807dd0db5631e711cb24de1e52cb7a95d1cc/homeassistant/const.py#L882

Why do we call it parts per cubic meter?